lib/charms/mysql/v0/mysql.py

```diff
@@ -2552,7 +2552,9 @@ def flush_mysql_logs(self, logs_type: MySQLTextLogs) -> None:
         """Flushes the specified logs_type logs."""
         flush_logs_commands = (
             f"shell.connect('{self.server_config_user}:{self.server_config_password}@{self.instance_address}')",
+            'session.run_sql("SET sql_log_bin = 0")',
```

Comment:
does this has any effect on anything other then bin logs?

Reply:
https://dev.mysql.com/doc/refman/8.0/en/set-sql-log-bin.html: suggests that it only disables bin-logging of the current session

Reply:
from docs, looks precisely as what was needed